common/recordings.md

```diff
@@ -100,6 +100,17 @@ You can specify a cloud storage location for your recording files in your templa
 
 The above message ensures that your configuration is successful now, and all your recordings will start collecting in your configured destination.
 
+
+## Google and Alibaba Cloud Storage for Recordings
```

Comment:
the heading level is one level higher than i think what it should be
![Screenshot 2023-03-21 at 1 57 52 PM](https://user-images.githubusercontent.com/284612/226552344-44a1f0d7-bda4-46c9-8357-791bd8cc780e.png)